doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_arcgis_featuretable_sink.md

```diff
@@ -1,4 +1,4 @@
-# <a name="top"></a>NGSIArcGisSink
```

Comment:
This file has been renamed. Review references to it in other parts of the documenation (with grep it should be easy), so links can be fixed.

At least I think that .mkdocs.yml should be adjusted.

Reply:
done